metricbeat/module/prometheus/collector/_meta/data.json

```diff
@@ -14,15 +10,15 @@
     },
     "prometheus": {
         "labels": {
-            "interval": "15s"
+            "listener_name": "http"
         },
         "metrics": {
-            "prometheus_target_interval_length_seconds_count": 1,
-            "prometheus_target_interval_length_seconds_sum": 15.000226176
+            "net_conntrack_listener_conn_accepted_total": 3,
```

Comment:
Is this change related?

Reply:
Yes, this is generated from the `testdata/docs.json` file. This reproduced the issue this change is fixing as they are two different metrics with the same labels.
Previous example didn't reproduce the issue because they were values of the same summary or histogram.